ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2561,5 +2561,67 @@
       "name": "type-apache-camel.svg",
       "mediaType": "image/svg+xml"
     }
+  },
+  {
+    "id": "che7-python",
+    "name": "Che 7 Python",
+    "creator": "Anatolii Bazko",
+    "tags": [
+      "ws.next",
+      "Theia",
+      "Python",
+      "pip"
+    ],
+    "scope": "general",
+    "description": "Workspace.next sidecars and Theia as IDE",
+    "components": [
+      {
+        "version": "7",
+        "name": "Centos"
+      },
+      {
+        "version": "3.7.2",
+        "name": "Python"
+      },
+      {
+        "version": "19.0.3 ",
+        "name": "PIP"
+      }
+    ],
+    "workspaceConfig": {
+      "environments": {
+        "default": {
+          "recipe": {
+            "contentType": "application/x-yaml",
+            "type": "kubernetes",
+            "content": "kind: List\nitems:\n - \n  apiVersion: v1\n  kind: Pod\n  metadata:\n   name: ws\n  spec:\n   containers:\n    - \n     image: 'eclipse/che-dev:nightly'\n     name: dev\n     resources:\n      limits:\n       memory: 512Mi\n"
```

Comment:
You have only one single object here, and now it is not required to wrap it to List, so content may look like the following:
```yaml
apiVersion: v1
kind: Pod
metadata:
  name: ws
spec:
  containers:
  - 
    image: 'eclipse/che-dev:nightly'
    name: dev
    resources:
      limits:
        memory: 512Mi
```
```
apiVersion: v1\nkind: Pod\nmetadata:\n  name: ws\nspec:\n  containers:\n  - \n    image: 'eclipse/che-dev:nightly'\n    name: dev\n    resources:\n      limits:\n        memory: 512Mi\n
```